### DISCUSSION TITLE
How to disable link?

### DISCUSSION THREAD

**Discussion by uluumbch** [ash-jc-allen/short-url#167] [Q&A]
I want to disable link, which mean i want to edit that link so it's cannot visited again. just like single use concept, but i want to disable the link only if i want to.

How can i achieve that?

**Comment by ash-jc-allen** [comment]
Hey @uluumbch! Off the top of my head, I think you could update the `ShortURL` model's `deactivated_at` field. In the docs (https://github.com/ash-jc-allen/short-url#activation-and-deactivation-times) you can see how we set it when creating the short URL. So you might be able to do something similar to that?

E.g. - 

```php
$shortUrl = ShortURL::find(SHORT_URL_ID_HERE);

$shortUrl->update(['deactivated_at' => now()]);
```

Do you think something like this would solve your problem? 🙂
